CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Castle Core Changelog
 
+## Unreleased
+
+Bugfixes:
+- Make ProxyUtil.IsAccessible(MethodBase) take into account declaring type's accessibility so it doesn't report false negatives for e.g. public methods in inaccessible classes. (@stakx, #289)
+
```

Comment:
Some things are actually missing here:

1. The `ProxyUtil` class having become `static`. (Possibly a breaking change, even though it's very unlikely.)
2. Members of `InternalsUtil` now being obsolete.

Under which heading do you usually list changes that aren't breaking changes?